utils/decorators.py

```diff
@@ -12,12 +12,14 @@
 
 def lock_method_for_strangers(
         checker: Callable[[int, List[int]], bool],
+        stop_bloggers: Callable = None,
```

Comment:
Please, do not change this decorator. As I mentioned above, we should keep this original decorator and re-apply it with a different checker method instead. You can revert all changes in this file.